react-sortable-hoc/index.d.ts

```diff
@@ -1,6 +1,6 @@
 // Type definitions for nes 0.0.7
 // Project: https://github.com/clauderic/react-sortable-hoc
-// Definitions by: Ivo Stratev <https://github.com/NoHomey>
+// Definitions by: Andrew de Waal <https://github.com/andrewdewaal>
```

Comment:
Definitions by should be list of contributors contributed to the given Definition, please re-add my information as contributor! And please in the future do not remove contributor information just by changing one line ...